commander/android/src/main/kotlin/com/gojuno/commander/android/Adb.kt

```diff
@@ -90,11 +90,12 @@ fun connectedAdbDevices(): Single<Set<AdbDevice>> = process(listOf(adb, "devices
 
 fun AdbDevice.log(message: String) = com.gojuno.commander.os.log("[$id] $message")
 
-fun AdbDevice.installApk(pathToApk: String, timeout: Pair<Int, TimeUnit> = 2 to MINUTES): Observable<Unit> {
+fun AdbDevice.installApk(pathToApk: String, timeout: Pair<Int, TimeUnit> = 2 to MINUTES, logErrors: Boolean = false): Observable<Unit> {
```

Comment:
I see 3 variable names for this boolean - `verboseOutput`, `logErrors`, `print`. I have used `logErrors` for consistency with methods below:

https://github.com/trevjonez/composer-gradle-plugin/blob/5cf9ffafa02185cb82e30d3022d934774955925b/commander/android/src/main/kotlin/com/gojuno/commander/android/Adb.kt#L163

https://github.com/trevjonez/composer-gradle-plugin/blob/5cf9ffafa02185cb82e30d3022d934774955925b/commander/android/src/main/kotlin/com/gojuno/commander/android/Adb.kt#L178

Reply:
Could probably use a closer pass to unify the naming. seems like it is more often just a `print` boolean. I'm not worried about it, minutia.